### ISSUE TITLE
Add Device filter in making purchase order

### ISSUE THREAD

**Issue by rajeshdungrani** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#881]
Currently while adding item in PO we have option of SKU and Name (Item name). But it is difficult to remember SKU and in Name box all the items apear. Now if you can provide an addtional filter of Device (Model) before Name, it wud be more efficient. After selecting Device from the dropdown filter, it will display parts related to that model only.

![image](https://user-images.githubusercontent.com/12892217/62041744-05571e00-b21a-11e9-8b6a-17339b7a927f.png)

**Comment by Uzair-RepairDesk** [comment]
Hey @rajeshdungrani 

Thank you for sharing your idea. However, we have already given the "Suggestive Search" option to search item from "Name". If you don't remember an SKU/ID of the item you can easily search item from following 2 way after selecting a supplier of an item:-

1: You can start typing relevant keywords of an item under the name and it gives you a list of the item along with Manufacturer and Device so you can select it and it automatically populates the SKU/ID of an item
![search](https://user-images.githubusercontent.com/22955861/62047244-6e8f5f00-b223-11e9-8c64-97026a77e45c.png)

2: You can print barcode of items. keep it and simply scan it by keeping focus under SKU/ID field so it will bring the same item in a row.

**Comment by rajeshdungrani** [comment]
Yes. That suggestive search is working beautifully. But we have a different concern in that. In India we are catering more than 15 Brands. Few Indian brands like Micromax, Lava etc have more than 200 models which are very difficult to remember. Suggestive search is very good, no doubt about it. But in our case, if we had option of selecting device (As we get option while raising new ticket from Manage Ticket), things would become more simpler and chances of error will reduce. It will give more operational convenience.

**Comment by Uzair-RepairDesk** [comment]
Hi @rajeshdungrani 

Thank you for your response and for sharing your concern. I can certainly see how suggested improvements would be useful specifically for you. Currently, we have to pick new features based on the value they’ll add to the most users possible. 

I am adding your request under enhancements so we can discuss it in-house in next quarter plan. Thanks
